fea-rs/src/bin/compile.rs

```diff
@@ -92,6 +109,26 @@ struct Args {
     #[arg(short, long, group = "glyph_source")]
     font: Option<PathBuf>,
 
+    /// Variable fonts only: a path to a file containing info on variation axes.
+    ///
+    /// Note that we can not correctly compile variable fonts, because we do not
+    /// have the ability to compile deltas. This functionality is provided for
+    /// debugging.
+    ///
+    /// This should be a utf-8 encoded file containing a list of axes and their
+    /// (min, default, max) values, in user coordinates.
+    ///
+    /// Blank lines and lines beginning with '#' will be skipped.
+    ///
+    /// e.g. it might look like,
+    ///
+    /// ```
+    /// wght 100 400 900
+    /// wdth 50 100 200
+    /// ```
+    #[arg(short, long)]
+    axis_info: Option<PathBuf>,
```

Comment:
Why do we insist on a pathbuf, seems like it could be passed directly w/o much issue unless you have a very large axis set, e.g. `--axis-info "wght 100 400 900, wdth 50 100 200"

Reply:
I considered that, and then I considered both, but since this is intended for debugging I figure it would make sense to just write a file out with the appropriate axis info alongside `features.fea`.